KYHBPA_TeamA/KYHBPA_TeamA/Views/Home/Index.cshtml

```diff
@@ -45,7 +45,7 @@
 
 
 <section id="primary">
-    <h1 class="common-Uppercase">Kentucky Horsemen's <br> Benevolent and Protective <br> Association</h1>
+    <h1 class="common-Uppercase">Kentucky Horsemen's Benevolent and Protective <br> Association</h1>
```

Comment:
<h1 class="common-Uppercase">Kentucky Horsemen's Benevolent <br>and Protective Association</h1>

Can you break here instead? I think it looks better